stub/src/main/java/io/grpc/stub/CallStreamObserver.java

```diff
@@ -106,9 +106,33 @@
    *   </li>
    * </ul>
    * </p>
+   *
+   * @deprecated Use {@link #disableAutoRequest} instead. This method will be removed.
```

Comment:
Mention that they will need one additional `request(1)` to match the new behavior. It might be worth putting that as a code snippet in the longer method javadoc.

Reply:
Added a note on migrating with an snippet of example code to the javadoc of disableAutoInboundFlowControl.  Also called out that for server-side, no change is required, but for client side an additional request is needed.